### ISSUE TITLE
shorthand-values rule fails to identify even more negative values

### ISSUE THREAD

**Issue by alfaproject** [sasstools/sass-lint#424]
This is related to #375 

```css
test {
    margin: 0 10px 0 -10px;
}
```

**Comment by DanPurdy** [comment]
HI @alfaproject what is the error you're seeing? Can we have a little more information here please.

This can't be written as shorthand and isn't giving me any errors or warnings in `1.3.3`

Have you updated to the latest version?

**Comment by alfaproject** [comment]
Sorry, my bad. The Atom plugin is not using the latest version yet. ):

**Comment by DanPurdy** [comment]
Hi @alfaproject I'm the developer of the atom plugin. If you u install and reinstall it, it will be using the latest version you just need to bump the packages included in it as it's set to always use the latest version of 1.x

**Comment by alfaproject** [comment]
Indeed, I forgot to do it before reporting. Sorry about that.
